Unit Testing/src/main/Calculator.java

```diff
@@ -0,0 +1,123 @@
+public class Calculator {
+    private Integer a, b, c;
+
+    public Calculator(Integer a, Integer b, Integer c) {
+        this.a = a;
+        this.b = b;
+        this.c = c;
+    }
+
+    public Calculator(Integer a) {
+        this.a = a;
+    }
+
+    public Calculator() {
+    }
+
+    public Integer getA() {
+        return a;
+    }
+
+    public void setA(Integer a) {
+        this.a = a;
+    }
+
+    public Integer getB() {
+        return b;
+    }
+
+    public void setB(Integer b) {
+        this.b = b;
+    }
+
+    public Integer getC() {
+        return c;
+    }
+
+    public void setC(Integer c) {
+        this.c = c;
+    }
+
+    public Integer getMax() {
+       if (a==null) {
+           a = Integer.MIN_VALUE;
+       }
+       if (b==null) {
+           b = Integer.MIN_VALUE;
+       }
+        if (c==null) {
+           c = Integer.MIN_VALUE;
+        }
+        if (a > b && a > c)
+            return a;
+        if (b > a && b > c)
+            return b;
+        return c;
+    }
+
+    public Integer getMin() {
+        if (a==null) {
+            a = Integer.MIN_VALUE;
```

Comment:
Daca pui Integer.MIN_VALUE ca valoare default atunci cand calculezi minimul vei avea ca rezultat Integer.MIN_VALUE . In cazul asta ai putea seta valoarea cu MAX_VALUE